public/js/template.js

```diff
@@ -0,0 +1,38 @@
+jQuery(document).ready(function($) {
+
+	var my_nav = $('.navbar-sticky'); 

```

Comment:
Codacy found an issue: [Identifier 'my_nav' is not in camel case.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6084628)